### PR TITLE
implemented getPlatformVersion

### PR DESCRIPTION
**Pull request by scottdixon**
Part of `appium-android-driver` getSession fix.

@imurchie

### CODE REVIEW

File: lib/tools/adb-commands.js [hunk 1]
```diff
@@ -35,6 +35,15 @@ methods.getApiLevel = async function () {
   }
 };
 
+methods.getPlatformVersion = async function () {
+  log.info("Getting device platform version");
+  try {
+    return this.shell(['getprop', 'ro.build.version.release']);
```
Comment (jlipps) [inline lib/tools/adb-commands.js]: our new style guide says this should be: `return await this.shell(['getprop', 'ro.build.version.release']);`

**Comment by jlipps** [comment]
LGTM! would be nice to have an e2e test as well

**Comment by imurchie** [comment]
LGTM.

**Comment by scottdixon** [comment]
updated style + created e2e @jlipps

**Comment by jlipps** [comment]
looks great!

**Pull request merged**